src/clr-angular/utils/popover/popover-content.ts

```diff
@@ -82,7 +82,9 @@ export class ClrPopoverContent implements AfterContentChecked, OnDestroy {
       this.checkCollector.pipe(debounceTime(0)).subscribe(() => {
         this.alignContent();
         this.shouldRealign = false;
-        this.renderer.setStyle(this.view.rootNodes[0], 'opacity', '1');
+        if (this.view) {
```

Comment:
This looks right, but might as well guard against it on line 78 above as well. See https://github.com/vmware/clarity/pull/4286/files#diff-f1bb4a055321e60324934fdfc284ef85

Reply:
Okay, added the check